src/api/draftSpecs.ts

```diff
@@ -65,3 +98,7 @@ export const generateDraftSpec = (
 
     return draftSpec;
 };
+
+export const deleteDraftSpec = (draftId: string) => {
```

Comment:
This API was added in an earlier iteration and is no longer in use for edit. It is preserved for future use with the upcoming collection work.